scripts/install.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+
+python -m venv venv
+source venv/bin/activate
+
+pip install --upgrade pip
+pip install -e .
+
+mkdir /etc/auditor-apel-plugin
+
+ln -s $PWD/apel_plugin.cfg /etc/auditor-apel-plugin/apel_plugin.cfg
```

Comment:
I noticed it here, but this comment is more general: Sometimes you use `"apel_plugin"` and sometimes `"auditor_apel_plugin"`. Should we be more consistent?

Reply:
Yes, the naming should be discussed at some point and will then be adjusted.